backend/inspirehep/records/api/literature.py

```diff
@@ -171,6 +171,22 @@ def update(self, data, disable_orcid_push=False, *args, **kwargs):
                 push_to_orcid(self)
             self.push_authors_phonetic_blocks_to_redis()
 
+    def get_modified_authors(self):
+        previous_authors = get_value(self._previous_version, "authors", [])
```

Comment:
if you're not using any "advanced" features of `get_value` (there is no `.` or `[...]` in the second argument), better use the `get` method which is faster.